dpnp/backend/extensions/lapack/gesv.cpp

```diff
@@ -65,6 +65,11 @@ static sycl::event gesv_impl(sycl::queue exec_q,
                              std::vector<sycl::event> &host_task_events,
                              const std::vector<sycl::event> &depends)
 {
+#if defined(USE_ONEMKL_INTERFACES)
+    // Temporary flag for build only
+    // FIXME: Need to implement by using lapack::getrf and lapack::getrs
+    std::logic_error("Not Implemented");
```

Comment:
I believe `future_error` sounds more proper here
```suggestion
    std::future_error("Not Implemented");
```

Reply:
`std::future_error` defined for special std libraries like std::future, std::promise, etc. Function required `future_error_code` as input argument. 

Will not be replaced.